python/lsst/validate/drp/validate.py

```diff
@@ -200,9 +206,25 @@ def run(repo, visitDataIds, good_mag_limit=21.0,
         Expected photometric RMS [mmag] across visits.
     matchRef : int, optional
         Expectation of the number of stars that should be matched across visits.
+    makePrint : bool, optional
+        Print calculated quantities (to stdout).
+    makePlot : bool, optional
+        Create plots for metrics.  Saved to current working directory.
+    makeJson : bool, optional
+        Create JSON output file for metrics.  Saved to current working directory.
+    outputPrefix : str
+        Specify the beginning filename for output files.
+
+    Outputs
```

Comment:
Python arguments should go in a section named `Parameters` instead of `Inputs` and since the function doesn't return any argument there isn't a `Returns` field.

I'd put the 'Outputs' content in the Extended summary (which is just the paragraphs that follow the one-sentence summary at the top of the docstring).

See http://developer.lsst.io/en/latest/docs/py_docs.html#common-structure-of-docstrings for sections in Numpydoc-formatted docstrings.